apps/avifgainmaputil/program_command.h

```diff
@@ -44,6 +44,65 @@ class ProgramCommand {
   std::string description_;
 };
 
+// avifPixelFormat converter for use with argparse.
+// Actually converts to int, converting to avifPixelFormat didn't seem to
+// compile.
+struct PixelFormatConverter {
+  argparse::ConvertedValue<int> from_str(std::string str);
+  std::vector<std::string> default_choices();
```

Comment:
Camel case intended?

Reply:
Yes, these are the methods expected by argparse.